### PR TITLE
rename methods for command buffer to methods for command encoder

### LINKED ISSUE #250: WGPUCommandBufferId vs. WGPUCommandEncoderId
Should this not be WGPUCommandEncoderId?

https://github.com/gfx-rs/wgpu/blob/32399cff8a0b55389f2d2493ea5c900c986c2547/ffi/wgpu.h#L647

### PR DESCRIPTION
**Pull request by porky11**
closes #250

### CODE REVIEW

File: ffi/wgpu-remote.h [hunk 1]
```diff
@@ -1,72 +0,0 @@
-
```
Comment (grovesNL) [inline ffi/wgpu-remote.h]: Looks like this was accidentally removed, maybe it wasn't regenerated during `make`? If you can't get this to be generated, we could just amend the commit to keep it

Comment (porky11) [reply ffi/wgpu-remote.h]: Yeah, something doesn't work correctly with ffi creation using make, not sure why. I'll fix it and try it again

**Comment by porky11** [comment]
Should be fixed now

**Comment by kvark** [comment]
ah, didn't realize we have a conflict

**Comment by kvark** [comment]
@porky11 please rebase

**Comment by kvark** [comment]
Replaced by #299

**Pull request closed without merging**